docs/asyncapi.yaml

```diff
@@ -86,61 +85,59 @@ components:
 
     crn:
       type: number
-      description: Customer reference Number
+      description: Customer Reference Number (consists of 10 digits)
       minimum: 1050000000
       maximum: 9999999999
       example: 1234567890
 
     sbi:
       type: number
-      description: Single Business Identifier
+      description: Single Business Identifier (consists of 9 digits)
       minimum: 105000000
       maximum: 999999999
       example: 123456789
 
     sourceSystem:
       type: string
-      description: The name of the source system that originated the message
+      description: Name of the source system that originated the message
       example: "AHWP"
       minlength: 3
       maxlength: 100
       pattern: "^[a-z0-9-_]+$"
 
     notifyTemplateId:
       type: string
-      description: Notification template identifier
+      description: Notify template identifier
       example: "f33517ff-2a88-4f6e-b855-c550268ce08a"
       format: uuid
 
     commsType:
       type: string
-      description: Type of communication (e.g., email, SMS)
+      description: Type of communication (e.g. email, SMS)
       default: "email"
       example: "email"
       enum: ["email"]
 
-    commsAddress:
+    commsAddresses:
       oneOf:
         - type: string
-          description: The user's contact email address
+          title: Accepts a single string
+          description: A single contact email address corresponding to one recipient
           example: "mrMegaFarmer@strawberryFortFarm.co.uk"
           minLength: 1
           maxLength: 320
           pattern: "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}(\\.[a-zA-Z]{2,})?$"
-          title: accepts a single string
         - type: array
-          title: accepts an array of strings
-          items:
-            type: string
-            description: A list of contact email addresses
-            example: "mrMegaFarmer@strawberryFortFarm.co.uk"
-            minLength: 1
-            maxLength: 320
-            pattern: "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}(\\.[a-zA-Z]{2,})?$"
+          title: Accepts an array of strings
+          description: A list of contact email addresses corresponding to multiple recipients
+          example: ["mrMegaFarmer@strawberryFortFarm.co.uk", "msMegaFarmer@blueberryCottageFarm.co.uk"]
```

Comment:
Seems like we already had specification under the `commsAddresses` field that services could send the content of a single message/request to multiple users. This field also originally said that it `accepts an array of strings` but only had a single string as the example. I have updated this to instead be an array that includes more than one email address.

Reply:
My only question is do we actually need a single string option? Will make it a lot easier for us if we only accept an array even if it only has one item.

Reply:
Sending only a string will work as the logic in the Comms microservice will wrap single strings in `[]` as if they were sent as an array, but agree that trying to explain this is maybe overkill and might cause confusion? Think in general if people just always wrap the email addresses in an array then there's always that awareness they can send the same message content to multiple users. I've updated the documentation to reflect this.

Reply:
This also updates the `Payload Example` to properly demonstrate that an array of multiple strings (email addresses) can be sent - so leaning more to the approach of simply defining `commsAddresses` as an array. Think it's worth noting as well that since as a team we agreed to rename it from singular to plural (`commsAddress` -> `commsAddresses`) that defining it as an array from the start makes the most sense (in my opinion).